src/sandbox/functionsSimulations/index.ts

```diff
@@ -1,87 +1,15 @@
 import * as functionsToolkit from "@chainlink/functions-toolkit";
```

Comment:
Do wen need this functionsSimulations wrapper anymore? we could just use functionsToolkit.simulateResponse() directly? this seems like an avoidable abstraction

Reply:
Yes, we can simplify it removing the abstraction layer here. I just want to implement all the functionality in one way

Reply:
as per the comment [here](https://github.com/smartcontractkit/hardhat-chainlink/pull/11#discussion_r1450061142) we can leave the thin wrappers since end users will not access the simulator  directly via the toolkit but indirectly via this plugin.